recipes/llvm-openmp-mutex/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "llvm-openmp-mutex" %}
+{% set version = "4.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  path: .
+
+build:
+  number: 1
+  skip: true  # [win and vc<14]
+
+outputs:
+  - name: _openmp_mutex
+    script: build-openmp_impl.sh
+    version: {{ openmp_ver }}
+    requirements:
+      host:
+        - llvm-openmp >=9.0.1
+      run:
+        - llvm-openmp >=9.0.1
+        # the selector here serves no real purpose, but conda appears
+        # to error without it, so here we are
+        - _libgcc_mutex {{ _libgcc_mutex }}  # [linux]
+    build:
+      string: 0_llvm
+      skip: True  # [not linux]
+      always_include_files:
+        # this major version has to match the libgomp version from gcc
+        - lib/libgomp.so.1
+      track_features:
+        - llvm_openmp
```

Comment:
```suggestion
        - llvm_openmp
      ignore_run_exports:
        - _openmp_mutex
```

Reply:
Huh? I thought we wanted run exports for the mutex packages per https://github.com/conda-forge/ctng-compilers-feedstock/pull/13

Reply:
`ignore_run_exports` is to make sure that the `llvm-openmp` doesn't add `_openmp_mutex` on this package itself. Otherwise `llvm-openmp` will add a dependency of `_openmp_mutex` on `_openmp_mutex` making a cycle.

Reply:
Ack yes sorry.